pkgs/applications/blockchains/nimbus/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  stdenv, fetchgit, lib, nim, which, writeScriptBin,
+  # Options: nimbus_light_client, nimbus_validator_client, nimbus_signing_node
+  makeTargets ? [ "nimbus_beacon_node" ],
+  # WARNING: CPU optmizations that make binary not portable.
+  nativeBuild ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nimbus";
+  version = "22.11.0";
+  commit = "356dd4ee";
+  name = "${pname}-${version}-${commit}";
```

Comment:
```suggestion
```

name should generally be pname+"-"+version. Also we do not need a git short sha, it is usually not worth the extra effort to make it compatible with udpate scripts.

Reply:
But I need the commit for the sake of `git rev-parse` being called by the build process, since I want the commit included in the program name. This is actually kinda important for tracking exact versions of nodes running on the network. It stays.